demo/summer-2016.html

```diff
@@ -214,6 +214,7 @@
 <script src="../build/src-noconflict/ext-language_tools.js"></script>
 <script>
     ace.require("ace/ext/language_tools");
+    smth = ace.require("ace/lib/katex/katex");
```

Comment:
`smth` is not the best name for the module. By the way, pls log a message to the console saying that you've loaded katex successfully and printing this `smth`